components/tooltip/tooltip.directive.ts

```diff
@@ -33,6 +34,8 @@ export class TooltipDirective {
   private visible:boolean = false;
   private tooltip:ComponentRef<any>;
 
+  private delayTimeoutId:NodeJS.Timer;
```

Comment:
Ok then... What do we do with CI error? If I change to number, the build will fail